spec/job_spec.rb

```diff
@@ -37,6 +37,21 @@
     end
   end
 
+  context '#job_plays' do
+    let(:url)            { "example.com/api/v1/job_plays" }
+    let(:job_collection) { build(:response_collection, :klass => described_class) }
+    let(:job_events)     { build(:response_url_collection, :klass => AnsibleTowerClient::JobEvent, :url => url) }
+    it "returns a collection of AnsibleTowerClient::JobEvents" do
```

Comment:
@bzwei -Not sure how we should handle this - The Gem is doing the right thing - `job_plays` return data of `type` = `job_event` Here is a sample directly from the Tower API:

```
GET /api/v1/jobs/56/job_plays/
HTTP 200 OK
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept
X-API-Time: 0.040s




{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "play": "Hello World Sample",
            "failed_count": 0,
            "changed_count": 0,
            "started": "2016-11-29T21:34:42.175Z",
            "changed": false,
            "related": {
                "job_event": "/api/v1/job_events/151/"
            },
            "skipped_count": 0,
            "failed": false,
            "ok_count": 2,
            "unreachable_count": 0,
            "type": "job_event",
            "id": 151
        }
    ]
}
```